cli/src/main/java/com/box/l10n/mojito/cli/command/PullCommand.java

```diff
@@ -62,6 +60,11 @@
     @Parameter(names = {Param.SOURCE_REGEX_LONG, Param.SOURCE_REGEX_SHORT}, arity = 1, required = false, description = Param.SOURCE_REGEX_DESCRIPTION)
     String sourcePathFilterRegex;
 
+    @Parameter(names = {"--inheritance-mode"}, required = false, description = "How to handle missing translations "
```

Comment:
description -> `Inheritance Mode. Used when there is no translations in the target locale for a text unit. (USE_PARENT to fallback to parent locale translation, REMOVE_TRANSLATED to remove the text unit from the file)`